.github/ISSUE_TEMPLATE

```diff
@@ -0,0 +1,16 @@
+Please answer these questions before submitting your issue.
```

Comment:
I suggested the change because it makes it clearer that the information is only required for a subset of issues.